api/app.py

```diff
@@ -0,0 +1,140 @@
+from flask import Flask,request
+from flask_restplus import Api,Resource,fields
+from  werkzeug.security import generate_password_hash , check_password_hash
+from models import user
+
+
+app = Flask(__name__)
+api = Api(app)
+userObj = user()
+
+
+
+#start of api model
+registerModel = api.model('registerModel',{
+	"firstName":fields.String,
+	"lastName":fields.String,
+	"email":fields.String,
+	"password":fields.String,
+	"confirmPassword":fields.String,
+	})
+loginModel=api.model('loginModel',{
+	"email":fields.String,
+	"password":fields.String
+	})
+logoutModel=api.model('logoutModel',{
+	"logout":fields.String
+	})
+passwordResetModel=api.model("passwordResetModel",{
+	"oldPassword":fields.String,
+	"newPassword":fields.String,
+	"confirmPassword":fields.String
+	})
+registerBusinessModel=api.model("registerBusinessModel",{
+	'title':fields.String,
+	'description':fields.String,
+	'author':fields.String,
+	'businessId':fields.Integer
+	})
+updateBusinessModel=api.model('updateBusinessModel',{
+	'title':fields.String,
+	'description':fields.String,
+	})
+businessModel=api.model('businessModel',{
+	'businessId':fields.Integer
+	})
+#end of api model
+
+
+#user registration endpoint
+@api.route('/api/auth/register')
+class register(Resource):
+	@api.expect(registerModel)
+	def post(self):
+		newUser=api.payload
+		#email=str(request.data.get('email',''))
+		#email = newUser['email']
+		if newUser['password'] != newUser['confirmPassword']:
+		 	return {"result":"password do not match"}
+		elif userObj.findIn(userObj.users,newUser['email']) == True:
+		 	return {"result":"email already exist"}
+		elif userObj.emailCheck(newUser['email'])==False:
+			return {"result":"mast be a valid email"}
+		else:
+		 	newUser['password']=generate_password_hash(newUser['password'],"sha256")
+		 	del newUser['confirmPassword']
+		 	userObj.users.append(newUser)
+		 	return {"result":"added to database"}
+	def get(self):
+		return {'test':'123'}
+
+#user login end point 
+@api.route('/api/auth/login')
+class login(Resource):
+	@api.expect(loginModel)
+	def post(self):
+		pass
+
+
+#user logout endpoint
```

Comment:
Kindly install pylint, you  are clearly not following the pep8 standard guidelines